dev/lint-python

```diff
@@ -61,13 +79,28 @@ else
 fi
 
 if [ "$lint_status" -ne 0 ]; then
-    echo "Python lint checks failed."
+    echo "PEP8 checks failed."
     cat "$PYTHON_LINT_REPORT_PATH"
 else
-    echo "Python lint checks passed."
+    echo "PEP8 checks passed."
 fi
 
-# rm "$PEP8_SCRIPT_PATH"
 rm "$PYTHON_LINT_REPORT_PATH"
 
+
+for to_be_checked in "$PATHS_TO_CHECK"
+do
+    pylint --rcfile="$PYLINT_HOME/pylintrc" $to_be_checked >> "$PYLINT_REPORT_PATH"
```

Comment:
Really minor point, but I wonder whether we should put the `pylintrc` file in the root folder of the project and should name it `.pylintrc` so that it's picked up by default if you run `pylint` from the root of the repo.  This probably isn't a huge deal, since I don't think that will work out of the box anyways without setting the PYTHONPATH, but it would be consistent with how pep8's `tox.ini` can be found there, along with the `scalastyle.xml` file.